packages/controllers/src/snaps/utils.ts

```diff
@@ -296,28 +295,45 @@ export function validateNpmSnapManifest(
 }
 
 /**
- * @param version1
- * @param version2
+ * Checks whether a SemVer version is greater than another.
+ *
+ * @param version1 - The left-hand version.
+ * @param version2 - The right-hand version.
+ * @returns `version1 > version2`.
  */
-export function gtVersion(version1: string, version2: string) {
+export function gtVersion(version1: string, version2: string): boolean {
   return gtSemver(version1, version2, { includePrerelease: true });
 }
 
 /**
- * @param version
- * @param versionRange
+ * Returns whether a SemVer version satisfies a SemVer range.
+ *
+ * @param version - The SemVer version to check.
+ * @param versionRange - The SemVer version range to check against.
+ * @returns Whether the version satisfied the version rnage.
  */
-export function satifiesVersionRange(version: string, versionRange: string) {
+export function satifiesVersionRange(
+  version: string,
+  versionRange: string,
+): boolean {
   return satifiesSemver(version, versionRange, {
     includePrerelease: true,
   });
 }
 
 /**
- * @param versions
- * @param versionRange
+ * Return the highest version in the list that satisfies the range,
```

Comment:
I wonder whether we should document the "use non-prerelease by default and fall back to prerelease if no matches found" strategy in the doc string or not.

Reply:
3f8379b